dashboard/src/shared/Secret.ts

```diff
@@ -0,0 +1,62 @@
+import axios from "axios";
+
+import { ISecret, IStatus } from "./types";
+
+export default class Secret {
+  public static async create(
+    name: string,
+    secrets: any,
```

Comment:
lets call this parameter `data` and give it type `{ [s: string]: string; }` since it cannot be a nested object

Reply:
cannot use `data` because of `const { data } = await axios.post<ISecret>(url, {` line in the function